scipy/optimize/_basinhopping.py

```diff
@@ -437,6 +437,19 @@ def basinhopping(func, x0, niter=100, T=1.0, stepsize=0.5,
         `take_step` and `accept_test`, and these functions use random
         number generation, then those functions are responsible for the state
         of their random number generator.
+    accept_rate : float, optional
```

Comment:
target_accept_rate would be more self-explanatory, but it is also longer, so whatever you feel is better is fine.

Reply:
I think `target_accept_rate` would be preferred here.